node/utils/products.ts

```diff
@@ -51,3 +51,9 @@ export const mapSkus = (skuURL: string) => {
     }, []) ?? []
   )
 }
+const decimalNumber = (num: number): string | number => {
+    if(num === undefined) return num
```

Comment:
@elkyndiosa it's a little confuse, if you receive an undefined value, so you gonna return an undefined response, and this doesn't match with your response types. btw: you can easily use `if (!num) return ...`